crates/biome_service/src/settings.rs

```diff
@@ -1459,36 +1436,6 @@ pub fn to_format_settings(
     })
 }
 
-impl TryFrom<OverrideFormatterConfiguration> for FormatterSettings {
```

Comment:
These `TryFrom` implementations were never used, but contribute to multiple sources of truth. So I deleted them.